src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/FieldPrefixFunction.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root  or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+package com.salesforce.dynamodbv2.mt.mappers.sharedtable.impl;
+
+import com.google.common.base.Splitter;
+import com.salesforce.dynamodbv2.mt.context.MTAmazonDynamoDBContextProvider;
+import org.apache.commons.lang.StringUtils;
+
+import java.util.List;
+
+/*
+ * @author msgroi
+ */
+class FieldPrefixFunction {
+
+    private final String delimiter;
+
+    FieldPrefixFunction(String delimiter) {
+        this.delimiter = delimiter;
+    }
+
+    FieldValue apply(MTAmazonDynamoDBContextProvider mtContext, String tableIndex, String value) {
```

Comment:
where is the code that turns a field value into a string?

Reply:
One level up, in FieldMapper.apply() ... https://github.com/salesforce/mt-dynamo/blob/aae37cb2ec8a99b144e48aab35a2fa95b946b29b/src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/FieldMapper.java#L37

Reply:
fixed some formatting ... f2da4a6fda9e241c1963f8d1710e86753e338f1c